templates/promo.html

```diff
@@ -0,0 +1,112 @@
+<div>
```

Comment:
Would be much easier just to rename the file to `.js`, replace `class` with `className` to return this as JSX. Then we won't need a custom build step here when we serve with webpack.